alacritty/src/cli.rs

```diff
@@ -62,8 +62,12 @@ impl Options {
     /// Build `Options` from command line arguments.
     pub fn new() -> Self {
         let mut options = Options::default();
-
-        let version_string = format!("{} ({})", crate_version!(), env!("GIT_HASH"));
+        let commit_hash = env!("GIT_HASH");
+        let version_string = if commit_hash.is_empty() {
+            format!("{}", crate_version!())
+        } else {
+            format!("{} ({})", crate_version!(), commit_hash)
+        };
```

Comment:
I'd consider moving this into a `version` function for `Options`.

Reply:
I also think that the following might be a bit cleaner:

```rust
let mut version = crate_version!().to_owned();
if !commit_hash.is_empty() {
    version = format!("{} ({})", version, commit_hash);
}
```